tests/sanity/.env

```diff
@@ -4,4 +4,6 @@ PLATFORM_USER='user1'
 PLATFORM_USER_SECOND='user2'
 PLATFORM_TOKEN='eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6InVzZXIxIiwid29ya3NwYWNlIjoic2FuaXR5LXdzIn0.hfUCqePHO-WNps2by4B-CYGKIpDpLG0WVCUUtU-SVI4'
 SETTING=storage.json
-SETTING_SECOND=storageSecond.json
\ No newline at end of file
+SETTING_SECOND=storageSecond.json
+
+LIVENESS_URI='https://front.hc.engineering'
```

Comment:
I don't see why we need a separate URI for these tests. Liveness tests are just a subset of all tests that run against some deployment. 
Ideally, we should be able to run them against local setup or CI as well. In order to be able to do this we need only two things:
1. Properly tag these tests so we can filter / filter them out
2. Provide separate tests configuration

An alternative would be to create separate folder with tests.

Reply:
this will be moved to uberflow repo, the grep tag is already there @livness which won't run any tests on CI that are marked with @livness... 
Basically the idea is exclude these types of tests from a regular run, but to create a separate Cron workflow which will check if the production is working or not every lets say 4 hours or once a day.

Reply:
But I will just modify this PR a bit so we have these tests cover our current tests as well

Reply:
@aonnikov ok the PR is now set for regular CI check